ReactiveCocoa/NSObject+KeyValueObserving.swift

```diff
@@ -10,15 +10,16 @@ extension Reactive where Base: NSObject {
 	///
 	/// - parameters:
 	///   - keyPath: The key path of the property to be observed.
+	///   - options: KVO options, see `NSKeyValueObservingOptions` for more documentation.
```

Comment:
I agree.

I had it in at first but removed it when I saw that the generated (jazzy) documentation included the default values anyway. I'll add it later, it doesn't add much noise but probably adds some clarity, especially if working with the quick-doc.